hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/schemas/InitialModServiceTokenSchema.java

```diff
@@ -343,9 +343,12 @@ public void migrate(@NonNull final MigrationContext ctx) {
                                         aliasesState
                                                 .get()
                                                 .put(new ProtoBytes(toAcct.alias()), toAcct.accountIdOrThrow());
-                                        if (toAcct.alias().toByteArray().length > 20) {
+                                        if (toAcct.alias().length() > 20) {
```

Comment:
Someday let's get rid of all of these `== 20` and `> 20` magic number relational expressions and replace them with something that self-describes as checking whether it is an evm address or not.

Reply:
True, we have a serviceable comment somewhere...